docs/common/low_power.md

```diff
@@ -0,0 +1,131 @@
+
+## Prerequisites
+
+If you are not familiar with cellular IoT fundamentals and low power concepts, it is highly recommended to complete the [Cellular IoT Fundamentals Developer Academy Course](https://academy.nordicsemi.com/courses/cellular-iot-fundamentals/) first. This course covers essential topics including:
+
+- LTE-M and NB-IoT network technologies basics
+- Power saving modes (PSM, eDRX)
+
+It is also recommended to read the white paper [Best Practices for Cellular Development](https://docs.nordicsemi.com/bundle/nwp_044/page/WP/nwp_044/intro.html).
+
+## Measuring Power Consumption
+
+### Power Profiling Tools
+
+Understanding your device's actual power consumption is critical for optimizing battery life. The following tools are recommended:
+
+#### Online Power Profiler for LTE
+
+For quick estimates and profiling of current consumption based on network and transmission parameters, use the [Online Power Profiler for LTE](https://devzone.nordicsemi.com/power/w/opp/3/online-power-profiler-for-lte).
+
+#### Power Profiler Kit 2 (PPK2)
+
+For accurate measurements and detailed profiling, check out the [PPK2: Power Profiler Kit 2](https://www.nordicsemi.com/Products/Development-hardware/Power-Profiler-Kit-2).
+For detailed guidance on how to use the PPK2, see the [Power Profiler Kit User Guide](https://docs.nordicsemi.com/bundle/ug_ppk2/page/UG/ppk/PPK_user_guide_Intro.html).
+
+## Application Power Configuration
+
+The Asset Tracker Template is configured with power saving features enabled by default. This section covers the key configuration options specific to the application.
+
+For general configuration of the LTE link, refer to the [LTE Link Control Documentation](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/modem/lte_lc.html).
+
+### PSM (Power Saving Mode)
+
+PSM is **enabled by default**. Configure TAU and RAT parameters in `prj.conf`:
+
+```config
+CONFIG_LTE_PSM_REQ_RPTAU_SECONDS=1800  # Periodic TAU: 30 minutes
+CONFIG_LTE_PSM_REQ_RAT_SECONDS=60      # Active Time: 1 minute
+```
+
+It's recommended to configure a Periodic TAU timer longer than the update interval of the application to avoid synchronization with the network between updates.
+
+See [Configuration guide - PSM](configuration.md#psm-power-saving-mode) for additional options. Note that actual values are negotiated with the network.
+
+### eDRX (Extended Discontinuous Reception)
+
+eDRX is **enabled by default**. Configure the eDRX interval in `prj.conf`:
+
+```config
+CONFIG_LTE_EDRX_REQ_VALUE_LTE_M="0000"  # eDRX interval 5.12 s for LTE-M
+CONFIG_LTE_EDRX_REQ_VALUE_NBIOT="0000"  # eDRX interval 5.12 s for NB-IoT
+```
+
+eDRX is used to periodically sleep during the Active Timer. With the default settings, the device will monitor for downlink data every 5.12 seconds for the duration of the Active Timer (60 seconds) in case the cloud sends data to the device.
```

Comment:
```suggestion
eDRX is used to periodically sleep during the PSM Active Timer. With the default settings, the device will monitor for downlink data every 5.12 seconds for the duration of the Active Timer (60 seconds by default) in case the cloud sends data to the device.
```